ast.c

```diff
@@ -287,11 +287,6 @@ static const char *closure_use_flags[] = {
 	AST_FLAG(MODIFIER_FINAL), \
 	AST_FLAG(MODIFIER_READONLY)
 
-static const char *visibility_flags[] = {
-	AST_VISIBILITY_FLAGS,
```

Comment:
Should probably also merge AST_VISIBILITY_FLAGS into AST_MODIFIER_FLAGS rather than having it as a separate macro?